From 6f97690bb3a37d701a67fdd8e0dbd4d8911c4d7c Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 11 Mar 2016 15:33:47 +0100
Subject: [PATCH] man: rtpr: add minimal manpage

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1316059
Upstream Status: iproute2.git commit e4ef49a465969
Conflicts: Context changes in Makefile due to missing other man pages.

commit e4ef49a465969a68704dbcfec43baa7f874f59ff
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Sep 18 20:17:05 2015 +0200

    man: rtpr: add minimal manpage

    While there is not much to explain about this rather trivial shell
    script, having a manpage for it serves as good point of reference for
    users wondering what it might be for.

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/Makefile |  2 +-
 man/man8/rtpr.8   | 25 +++++++++++++++++++++++++
 2 files changed, 26 insertions(+), 1 deletion(-)
 create mode 100644 man/man8/rtpr.8

diff --git a/man/man8/Makefile b/man/man8/Makefile
index 248ecfd..c60c773 100644
--- a/man/man8/Makefile
+++ b/man/man8/Makefile
@@ -1,6 +1,6 @@
 TARGETS = ip-address.8 ip-link.8 ip-route.8
 
-MAN8PAGES = $(TARGETS) ip.8 arpd.8 lnstat.8 routel.8 rtacct.8 rtmon.8 ss.8 \
+MAN8PAGES = $(TARGETS) ip.8 arpd.8 lnstat.8 routel.8 rtacct.8 rtmon.8 rtpr.8 ss.8 \
 	tc.8 tc-bfifo.8 tc-cbq.8 tc-cbq-details.8 tc-choke.8 tc-codel.8 \
 	tc-fq.8 \
 	tc-drr.8 tc-ematch.8 tc-fq_codel.8 tc-hfsc.8 tc-htb.8 \
diff --git a/man/man8/rtpr.8 b/man/man8/rtpr.8
new file mode 100644
index 0000000..5e32b2e
--- /dev/null
+++ b/man/man8/rtpr.8
@@ -0,0 +1,25 @@
+.TH RTPR 8 "18 September, 2015"
+
+.SH NAME
+rtpr \- replace backslashes with newlines.
+
+.SH DESCRIPTION
+.B rtpr
+is a trivial bash script which converts backslashes in standard input to newlines. It's sole purpose is to be fed with input from
+.B ip
+when executed with it's
+.B --oneline
+flag.
+
+.SH EXAMPLES
+.TP
+ip --onenline address show | rtpr
+Undo oneline converted
+.B ip-address
+output.
+
+.SH SEE ALSO
+.BR ip (8)
+
+.SH AUTHORS
+Stephen Hemminger <shemming@brocade.com>
-- 
1.8.3.1

